metecho/api/serializers.py

```diff
@@ -360,6 +360,14 @@ class TaskSerializer(serializers.ModelSerializer):
         queryset=Epic.objects.all(),
         pk_field=serializers.CharField(),
         serializer=EpicMinimalSerializer,
+        required=False,
+        allow_null=True,
+    )
+    project = serializers.PrimaryKeyRelatedField(
+        queryset=Project.objects.all(),
+        pk_field=serializers.CharField(),
+        required=False,
+        allow_null=True,
```

Comment:
@jerivas I think we also need an update to the `CaseInsensitiveUniqueTogetherValidator` below, which right now is enforcing that a task `name` is unique within a given `epic`. I'm not sure if now we want to enforce uniqueness within the parent `project`, or within the parent `project` _or_ `epic` (depending which the task has), or maybe not enforce at all because it's not important... @davisagli do you have an opinion on this?